### PR TITLE
Don't delete webpack.config.js on vaadinClean. Closes #43

### LINKED ISSUE #43: vaadinClean removes webpack config
I am not sure if this is a bug, because it seems fully intentional. But I would like to at least question the decision to remove webpack.config.js. It seems to me that this file is the place for user customisations. It says within the file:
```
 /**
  * This file has been autogenerated as it didn't exist or was made for an older incompatible version.
  * This file can be used for manual configuration will not be modified if the flowDefaults constant exists.
  */
```
So to me it seems wrong to remove this file on "clean". It is initially generated, but it is not a build artefact. It should be enough to remove the webpack.generated.js file.

### PR DESCRIPTION
**Pull request by mvysny**

### CODE REVIEW

File: src/main/kotlin/com/vaadin/gradle/VaadinCleanTask.kt [hunk 1]
```diff
@@ -45,7 +45,8 @@ open class VaadinCleanTask : DefaultTask() {
         project.delete("${project.projectDir}/node_modules",
                 "${project.projectDir}/package.json",
                 "${project.projectDir}/package-lock.json",
-                "${project.projectDir}/webpack.config.js",
+                // don't delete webpack.config.js: https://github.com/vaadin/vaadin-gradle-plugin/issues/43
```
Comment (caalador) [inline src/main/kotlin/com/vaadin/gradle/VaadinCleanTask.kt]: I would expect to not need to keep the commented out code in here.

Comment (mvysny) [reply src/main/kotlin/com/vaadin/gradle/VaadinCleanTask.kt]: It's merely for the sake of warning. I'd advise to keep the line referencing https://github.com/vaadin/vaadin-gradle-plugin/issues/43, but we can delete the commented-out follow-up line if you'd like.

Comment (caalador) [reply src/main/kotlin/com/vaadin/gradle/VaadinCleanTask.kt]: Perhaps add a Doc for the whole clean method that mentions that it shouldn't be removed and remove the inline commented lines. 

Also we when this supports 14.2 we should also see to remove `package-lock.yaml` and `pnpm-file.js` should I make a ticket about that?

Comment (mvysny) [reply src/main/kotlin/com/vaadin/gradle/VaadinCleanTask.kt]: Nah, it's such a tiny change. I'll update this PR, just a sec :)

**Pull request merged**